conf/countries.json

```diff
@@ -13,7 +13,7 @@
   {"code": "AT", "state":  "valid", "description": "Austria"},
   {"code": "AU", "state":  "valid", "description": "Australia"},
   {"code": "AW", "state":  "valid", "description": "Aruba"},
-  {"code": "AX", "state":  "valid", "description": "Åland Islands"},
+  {"code": "AX", "state":  "valid", "description": "Aland Islands"},
   {"code": "AZ", "state":  "valid", "description": "Azerbaijan"},
   {"code": "BA", "state":  "valid", "description": "Bosnia and Herzegovina"},
   {"code": "BB", "state":  "valid", "description": "Barbados"},
```

Comment:
Missing updates for:
* `Bonaire, Sint Eustatius and Saba`
* `Myanmar (Burma)`

Reply:
Amended cheers mate no idea how I missed those